### PR TITLE
Update Travis Docker file for Trusty to build gRPC code

### PR DESCRIPTION
**Pull request by cschuet**
[RFC=0002](https://github.com/googlecartographer/rfcs/blob/master/text/0002-cloud-based-mapping-1.md)

### CODE REVIEW

**Comment by cschuet** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user cschuet. Merge queue now has a length of 1.

File: scripts/install_grpc.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# Copyright 2016 The Cartographer Authors
```
Comment (gaschler) [inline scripts/install_grpc.sh]: 2017

**Comment by wally-the-cartographer** [comment]
Refusing to merge, since the combined checks state is failure

**Comment by cschuet** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user cschuet. Merge queue now has a length of 1.

**Pull request merged**